core/arch/arm/plat-stm32mp1/main.c

```diff
@@ -40,6 +41,10 @@ register_phys_mem_pgdir(MEM_AREA_IO_SEC, GIC_BASE, GIC_SIZE);
 
 register_ddr(DDR_BASE, CFG_DRAM_SIZE);
 
+#ifdef CFG_WITH_PAGER
+const struct clk_ops stm32mp1_clk_ops __rodata_dummy;
```

Comment:
for fixup commit 
> ```
> [review] plat-stm32mp1: register to clock framework"
>
> Makes clock operators structure const with __rodata_unpaged attribute.

my apologies. I don't understand how I could have tested that! Must be with pager disabled :(, this can't work. Dummy references must be filtered out at link stage.

@jenswi-linaro & @jforissier, would you mind fgiving your feedback on the added commit
`core: link: platform can define pager dummy resources`
to allow platforms to define pager dummy references.